cfme/fixtures/pytest_selenium.py

```diff
@@ -1137,7 +1150,11 @@ def _login_func():
                 'service evmserverd status').output)
             raise
 
-    if recycle:
+    if restart_evmserverd:
+        store.current_appliance.ssh_client.run_command("service evmserverd restart")
```

Comment:
1) Don't we already have a "restart" method for this which does the wait or at least means you don't need to send raw commands?
2) Is it a good idea to perform a server restart inside of a nav? What conditions cause this?